opentelemetry-api/src/opentelemetry/_metrics/instrument.py

```diff
@@ -87,45 +74,11 @@ class Asynchronous(Instrument):
     def __init__(
         self,
         name,
-        callback: TCallback,
-        *args,
+        callback,
```

Comment:
Can you add the type back? Those should live in the API still

Reply:
Ah, sure, sorry, removed it by mistake. I have added it back. I noticed that there was another symbol named `InstrumentT` in the API `instrument.py` module. I renamed `TCallback` to `CallbackT` to be consistent. Should the types be named `XT` or `TX`? :thinking:

Reply:
Shoudl be consistent. I think it was `XT` is what we are using elsewhere. You may need to update [here](https://github.com/open-telemetry/opentelemetry-python/blob/9ef6829b23a6bc59154ed0c644c72833d0a0e46f/docs/conf.py#L99-L102) if sphinx gives you issues

Reply:
Tests are passing so marking resolved